ragaai_catalyst/redteaming/red_teaming.py

```
@@ -0,0 +1,313 @@
+
+        
+    def run(
+        self,
+        description: str,
+        detectors: List[str],
+        response_model: Any,
+        examples: List[str] = [],
+        model_input_format: Dict[str, Any] = {
+            "user_input": "Hi, I am looking for job recommendations",
+            "user_name": "John"
+        },
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)